chart/templates/09-dashboards-helper.yml

```diff
@@ -26,7 +26,17 @@ spec:
 ---
 apiVersion: v1
 data:
+  # URL for connecting to OpenSearch Dashboards instance. When using Malcolm's internal
+  #   instance of OpenSearch Dashboards (i.e., OPENSEARCH_PRIMARY from opensearch.env
+  #   is 'opensearch-local' or 'opensearch-remote') this should be
+  #   'http://dashboards:5601/dashboards', otherwise specify the Dashboards URL
+  #   in the format 'protocol://host:port/uri'.
   DASHBOARDS_URL: "{{ include "malcolm.dashboardsurl" . }}"
+  # These values are used to handle the Arkime value actions to pivot from Arkime
```

Comment:
These values should actually be under the [98-nginx-proxy.yml](https://github.com/idaholab/Malcolm-Helm/pull/13/files#diff-fd8028abe5167c64f521e1ca379bdbe6b752397e0540c0fdbc9958cc6d119593) service. I agree it is not intuitive that they were included in ./config/dashboards.env, maybe for the non-kubernetes I'll move it to the nginx.env. But yeah, this needs to be moved to nginx-proxy.

Reply:
Fixed